masschallenge/django-accelerator
# [AC-9646] Business Proposition Field label Text change

**[AC-9646](https://masschallenge.atlassian.net/browse/AC-9646)**

Change of business proposition text on page.

**How to test:**

- Ensure that the definition of done according to [AC-9646](https://masschallenge.atlassian.net/browse/AC-9646) is satisfied.
